src/Microsoft.AspNetCore.Hosting.Abstractions/HostingAbstractionsWebHostBuilderExtensions.cs

```diff
@@ -160,6 +160,17 @@ public static IWebHostBuilder PreferHostingUrls(this IWebHostBuilder hostBuilder
         }
 
         /// <summary>
+        /// Specify if startup status messages should be written to console.
+        /// </summary>
+        /// <param name="hostBuilder">The <see cref="IWebHostBuilder"/> to configure.</param>
+        /// <param name="writeStatusMessages"><c>true</c> to enable writing of hosting startup status messages; otherwise <c>false</c>.</param>
+        /// <returns>The <see cref="IWebHostBuilder"/>.</returns>
+        public static IWebHostBuilder WriteStatusMessages(this IWebHostBuilder hostBuilder, bool writeStatusMessages)
```

Comment:
Ugh, does this really need to be first class?

Reply:
This method is optional, but several users have asked for it, it's not just a test feature.

Reply:
I know, I read the issues just like you, I *really* think we should avoid adding *more* of these methods and instead do https://github.com/aspnet/Hosting/issues/1042. It's much clearer and more discoverable.

Reply:
Spoke offline and this seems out of scope. We can explore alternatives separately.